draft-ietf-add-ddr.md

```diff
@@ -53,11 +53,13 @@ author:
 This document defines Discovery of Designated Resolvers (DDR), a
 mechanism for DNS clients to use DNS records to discover a resolver's encrypted
 DNS configuration. This mechanism can be used to move from unencrypted DNS to
-encrypted DNS when only the IP address of a resolver is known. This mechanism is
-designed to be limited to cases where unencrypted resolvers and their designated
-resolvers are operated by the same entity or cooperating entities. It can also be used
-to discover support for encrypted DNS protocols when the name of an encrypted resolver
-is known.
+encrypted DNS when only the IP address of a resolver is known.  This mechanism
+is designed to be limited to cases where unencrypted resolvers and the
+encrypted resolvers they designate to use in their own place, the "designated
+resolvers" for short, are operated by the same entity or cooperating entities.
```

Comment:
I don't like having this expansion / extra text in the abstract — I find it a bit harder to read and follow. Given that this document is about defining designated resolvers, it seems OK to mention the term in the abstract.

Reply:
Updated the text to move the definition to its own sentence earlier on.